config/initializers/rack_attack.rb

```diff
@@ -55,6 +55,15 @@ class Attack
 
         req.ip if (req.patch? || req.put?) && path_performs_event_sign_up
       end
+
+      throttle("event upsert", limit: 100, period: 1.day) do |req|
```

Comment:
As the API will already be rate limiting to 100req/day this isn't going to add any value; instead you would be better factoring in the client IP and dropping the rate limit to something like 5req/min/ip